plejd/ble.bluez.js

```diff
@@ -379,8 +377,20 @@ class PlejdService extends EventEmitter {
     this.characteristics.lastData.StartNotify();
   }
 
+  async throttledInit(delay) {
+    if(this.delayedInit){
+      return this.delayedInit;
+    }
+   this.delayedInit = new Promise((resolve) => setTimeout(async () => {
+     const result = await this.init();
+     this.delayedInit = null;
+     resolve(result)
+   }, delay))
+   return this.delayedInit;
+  }
+
   async write(data, retry = true) {
-    if (!this.plejdService || !this.characteristics.data) {
+    if (!data || !this.plejdService || !this.characteristics.data) {
```

Comment:
Apparently it is not unusual to get `!data` atleast with multiple Plejddevices and scenes.